### DISCUSSION TITLE
Quick inventory regardless of tags

### DISCUSSION THREAD

**Discussion by rawtalent06** [prowler-cloud/prowler#5141] [Ideas]
Are there any plans to allow an inventory to run that's shows everything with and without a tag? We have several resources in AWS that do not have tags but I want those to show up in the inventory. For instance, I have 18 lambda functions but only 6 are reported because they have tags. Ideally, I want this inventory to give me the correct amount. Maybe the better question is, is there a way to do a full inventory?

Below is from the Tutorials page.

    "The inventorying process is done with resourcegroupstaggingapi calls which means that only resources they have or have had tags will appear (except for the IAM and S3 resources which are done with Boto3 API calls)."

**Comment by jfagoagas** [comment]
Hello @rawtalent06 currently the AWS ResourceGroupsTagging API only knows about resources that have or have had tags. We are exploring other AWS APIs, and some other Cloud Providers, to improve the inventory but I cannot say more since we don't have any ETA for this.

I recommend you to open a Github issue in the repo where we can keep track of it.
